_includes/desktop-table.html

```diff
@@ -41,9 +41,13 @@
           </div>
         </td>
         <td class="twitter progress" colspan="6">
+          {% if website.twitter %}
           <a class="ui twitter mini button"
              href="https://twitter.com/share?url={{site.url|cgi_escape}}&amp;text={{progress_tweet|replace:'TWITTERHANDLE',website.twitter|cgi_escape}}&amp;hashtags={{page.hash|cgi_escape}}"
              target="_blank"><i class="twitter icon"></i> {{page.link_progress}}</a>
+          {% else %}
+          &nbsp;
```

Comment:
why the space?

Reply:
Since forever, some browsers go haywire when a table cell is empty (read: `<td></td>`) also see [here](https://www.emailonacid.com/blog/article/email-development/emailology_empty_table_cells/), so the space makes it display empty while not being considered empty by the browser.